samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/evaluation/TotalScoresBean.java

```diff
@@ -220,6 +232,53 @@ protected void init() {
 		
 		agents = newAgents;	
 	}
+
+  public boolean isOneSelectionType() {
+    if (this.getPublishedAssessment() != null) {
+      for (Object itemObject : this.getPublishedAssessment().getDefaultSection().getItemArray()) {
+        PublishedItemData item = (PublishedItemData) itemObject;
+        if (!(item.getTypeId().equals(TypeIfc.MULTIPLE_CHOICE)) && !(item.getTypeId().equals(TypeIfc.MULTIPLE_CORRECT_SINGLE_SELECTION)) && 
```

Comment:
Is item.getTypeId() a string, I feel it complex.
Try these methods equalsAny, EqualsNone, etc....

https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html#equalsAny-java.lang.CharSequence-java.lang.CharSequence...-

Reply:
It is a Long object and should be comprobate by equals method. But, if you want we can transform the Long object to String and comprobate them